### DISCUSSION TITLE
undo last action in multiplayer mode

### DISCUSSION THREAD

**Discussion by nobanzai** [bafolts/terraforming-mars#6383] [General]
Hi *,
is "undo last action" supposed to work in multiplayer mode?
It works as expected in solo mode, in multiplayer mode I can choose the option and get the appropriate radio button for the actions, but when selecting and confirming it, nothing happens.
I run the latest version of the server locally.
TIA.
Bye.
Michael.

**Comment by kberg** [comment]
More details are necessary. Are you hosting your own server? Was there a game? What actually happened?

**Comment by nobanzai** [comment]
As I wrote: "I run the latest version of the server locally."
What happened (always happens): I play a card, action is completed, I realize it was a mistake, I click "undo last action" and confirm it, but the action is not rolled back. Same works in solo mode.
I'm not sure, I understand your questions, though, because of course there was a game - how could I chose an undo action otherwise?
